crates/fluvio-connector-package/src/secret.rs

```diff
@@ -5,13 +5,25 @@ use std::{
     collections::HashSet,
 };
 
+use anyhow::{Result, anyhow};
+use lazy_static::lazy_static;
 use once_cell::sync::OnceCell;
+use regex::Regex;
 use serde::{Serialize, Deserialize, de::Visitor, Deserializer};
-use anyhow::{Result, anyhow};
-use serde_yaml::Value;
+use serde_yaml::{Value, Mapping, Sequence};
 
 static SECRET_STORE: OnceCell<Box<dyn SecretStore>> = OnceCell::new();
 
+lazy_static! {
```

Comment:
avoid lazy_static.  should use once_cell

Reply:
updated